apps/nextjs/src/app/admin/aila/[chatId]/view.tsx

```diff
@@ -2,25 +2,67 @@ import { useState } from "react";
 
 import type { AilaPersistedChat } from "@oakai/aila/src/protocol/schema";
 import { getSafetyResult } from "@oakai/core/src/utils/ailaModeration/helpers";
-import type { Moderation } from "@oakai/db";
-import { OakAccordion, OakPrimaryButton } from "@oaknational/oak-components";
+import type { Moderation, SafetyViolation } from "@oakai/db";
+import {
+  OakAccordion,
+  OakPrimaryButton,
+  OakFlex,
+  OakSmallSecondaryButton,
+  OakP,
+} from "@oaknational/oak-components";
 
 import { trpc } from "@/utils/trpc";
 
+function SafetyViolationItem({
+  safetyViolation,
+}: {
+  readonly safetyViolation: SafetyViolation;
+}) {
+  const { recordId } = safetyViolation;
+  const removeSafetyViolation = trpc.admin.removeSafetyViolation.useMutation(
+    {},
+  );
+  return (
+    <>
+      <OakFlex $alignItems="center" $background="pink" $pa="inner-padding-m">
+        <OakP $font="body-2" $mr="space-between-m">
+          <OakP $font="heading-7">
+            This moderation triggered a safety violation.
+          </OakP>{" "}
+          Reversing may unban a banned user
+        </OakP>
+        <OakSmallSecondaryButton
+          iconName="cross"
+          onClick={() =>
+            void removeSafetyViolation.mutateAsync({ recordId: recordId })
```

Comment:
For lines like this which could fail because of a network or serve error I've been adding sentry and toast

Reply:
But perhaps that's the wrong pattern? We should give user feedback in these two cases

Reply:
Good point. I put this together quite quickly so I didn't put too much thought into it. I'll look into that 👍

Reply:
Another option would be to render an error message in-line, but I don't think we have precedent for that

Reply:
I think for admin facing features doing a toast is probably fine and simplest for now